docs/dev/table/connectors/formats/debezium.md

```diff
@@ -185,6 +185,18 @@ Format Options
       <td>Skip fields and rows with parse errors instead of failing.
       Fields are set to null in case of errors.</td>
     </tr>
+    <tr>
+       <td><h5>json.timestamp-format.standard</h5></td>
```

Comment:
```suggestion
       <td><h5>debezium-json.timestamp-format.standard</h5></td>
```

Reply:
Plase also help to correct the above `json.ignore-parse-errors` to `debezium-json.ignore-parse-errors`.
The same to Canal docs.